deluge/common.py

```diff
@@ -961,24 +970,77 @@ def is_ipv6(ip):
 
     """
 
+    import socket
+
     try:
-        import ipaddress
-    except ImportError:
+        socket.inet_pton(socket.AF_INET6, ip)
+    except OSError:
+        return False
+    else:
+        return True
+
+
+def is_interface(interface):
```

Comment:
This is the wrong way round `is_interface` should be `is_adapter`. An interface is either an IP address or adapter id